cloudferrylib/os/actions/prepare_networks.py

```diff
@@ -31,6 +31,13 @@ def run(self, info=None, **kwargs):
         keep_ip = self.cfg.migrate.keep_ip
 
         instances = info_compute[utl.INSTANCES_TYPE]
+        # disable DHCP in all subnets
+        subnets = network_resource.get_subnets()
+        for snet in subnets:
+            if snet['tenant_name'] == self.cfg.dst.tenant:
```

Comment:
We will need to decide which keywords are allowed, then update it. Right now, it matches the way the config.ini works.
For keywords, we may also want to allow regex or a '*' in it.